bin/test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let Promise = require('bluebird');
+let spawn = require('child_process').spawn;
+let fs = Promise.promisifyAll(require('fs'));
+let path = require('path');
+
+let cmd = path.resolve(__dirname, '..', 'node_modules', '.bin', '_mocha');
+let args = ['--compilers', 'ts:ts-node/register,tsx:ts-node/register'];
+
+let services = path.resolve('services');
+return fs.readdirAsync(services).then(folders => {
+  return Promise.each(folders, f => {
+    let dir = path.resolve(services, f);
+    return fs
+      .readdirAsync(dir)
+      .then(subfolders => {
+        if (subfolders.indexOf('test') > 0) {
```

Comment:
Should be `if (subfolders.indexOf('test') > -1)`. Look [here](https://www.w3schools.com/jsref/jsref_indexof.asp) for more details